### PR TITLE
[ICL]Pr vdenc hevc

### PR DESCRIPTION
**Pull request by fulinjie**

### CODE REVIEW

File: libavcodec/qsvenc_hevc.c [hunk 1]
```diff
@@ -237,6 +237,9 @@ static const AVOption options[] = {
     { "main",    NULL, 0, AV_OPT_TYPE_CONST, { .i64 = MFX_PROFILE_HEVC_MAIN    }, INT_MIN, INT_MAX,     VE, "profile" },
     { "main10",  NULL, 0, AV_OPT_TYPE_CONST, { .i64 = MFX_PROFILE_HEVC_MAIN10  }, INT_MIN, INT_MAX,     VE, "profile" },
     { "mainsp",  NULL, 0, AV_OPT_TYPE_CONST, { .i64 = MFX_PROFILE_HEVC_MAINSP  }, INT_MIN, INT_MAX,     VE, "profile" },
+#if QSV_HAVE_VDENC
+    { "low_power", "enable low power mode(experimental: many limitations by mfx version, BRC modes, etc.)", OFFSET(qsv.low_power), AV_OPT_TYPE_BOOL, { .i64 =  0 }, 0, 1, VE},
```
Comment (lizhong1008) [inline libavcodec/qsvenc_hevc.c:240]: @fulinjie ,please  check Carl's comment, no to hide this option.

Comment (fulinjie) [reply libavcodec/qsvenc_hevc.c:240]: I intended to match the behavior in VDENC H264.
If modified, I think same modification should be applied in H264 too.

Comment (lizhong1008) [reply libavcodec/qsvenc_hevc.c:240]: yes, please update H264 too to keep consistent.

**Comment by linjiefu** [comment]
https://patchwork.ffmpeg.org/patch/12448/

**Comment by fulinjie** [comment]
Low power mode for hevc qsv is merged, close.

**Pull request closed without merging**